### PR TITLE
Cumulative update

### PR DESCRIPTION
**Pull request by voron**
Updates according to new SoW

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,60 @@
...
+### Requirements
+* linux/macos terminal 
+* git
+* gcloud
+* kubectl (version from gcloud is ok)
+* helm
+* follow "Before you begin" part of [GCP manual](https://cloud.google.com/kubernetes-engine/docs/how-to/iam)
+
+### Deploy
+* Create k8s GKE two zone cluster, use at least [n1-highmem4 instances](https://cloud.google.com/compute/docs/machine-types#n1_machine_types)
```
Comment (medvedev1088) [inline README.md]: Please also include the command for creating a cluster.

File: example-values-parity.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+image:
```
Comment (medvedev1088) [inline example-values-parity.yaml]: To reduce duplication with charts/parity/values.yaml let's only leave the configuration that people will need to override in most cases e.g. externalLBp2pIP: 203.0.113.0 and externalLBp2pDiscoveryIP: 203.0.113.0, remove other config values and add a comment `# for other configuration options refer to charts/parity/values.yaml`. This way all default values will be in a single file charts/parity/values.yaml. In particular, let's remove the image section from here, parity, resources (not sure people usually need to override it?).

Similar for example-values-bitcoind.yaml. Let me know if it doesn't make sense.

File: charts/parity/values.yaml [hunk 3]
```diff
@@ -17,6 +17,15 @@ fullnameOverride: ""
 # Don't use spaces or special chars
```
Comment (medvedev1088) [inline charts/parity/values.yaml:18]: For the image, let's use the version from example-values-parity.yaml

```
  repository: parity/parity
  tag: v2.5.8-stable
  pullPolicy: IfNotPresent
```

File: example-values-bitcoind.yaml [hunk 4]
```diff
@@ -0,0 +1,42 @@
+image:
+  repository: arilot/docker-bitcoind
```
Comment (medvedev1088) [inline example-values-bitcoind.yaml]: Does it use this Dockerfile https://github.com/arilot/bitcoind/blob/master/Dockerfile?

File: charts/bitcoind/values.yaml [hunk 5]
```diff
@@ -47,6 +47,14 @@ resources:
     cpu: "3000m"
     memory: "12000Mi"
 
+securityContext:
```
Comment (medvedev1088) [inline charts/bitcoind/values.yaml:50]: Remove unused comments in this file, e.g.:

```
#  - 127.0.0.1/32
#  - 127.0.0.2/32
```

**Review by medvedev1088 — CHANGES_REQUESTED** [review]
Rename svc.yaml to service.yaml for consistency with parity chart, replace -svc suffix with -service suffix, add 
```
labels:
    chain: btc
```

**Comment by medvedev1088** [comment]
Please remove _deployment.yaml and _pvc.yaml.

File: example-values-bitcoind.yaml (hunk 4 above)
Comment (voron) [reply example-values-bitcoind.yaml]: No, it uses https://github.com/arilot/docker-bitcoind/blob/master/Dockerfile file

File: charts/bitcoind/values.yaml (hunk 5 above)
Comment (voron) [reply charts/bitcoind/values.yaml:50]: I wanna leave IPs as format example solely

File: charts/parity/values.yaml (hunk 3 above)
Comment (voron) [reply charts/parity/values.yaml:18]: Done

File: example-values-parity.yaml (hunk 2 above)
Comment (voron) [reply example-values-parity.yaml]: It makes sense for sure.

File: README.md (hunk 1 above)
Comment (voron) [reply README.md]: I've added separate gke.md file

File: charts/bitcoind/values.yaml (hunk 5 above)
Comment (medvedev1088) [reply charts/bitcoind/values.yaml:50]: ok.

**Pull request merged**